input/projects/why-join-wip.md

```diff
@@ -0,0 +1,51 @@
+
+## IP and legal
+
+Legal can be tricky. We can help. Projects in our organization will get assistance to organize and manage their IP assignments as well as infrastructure for tracking common, simple, Contribution License Agreements across all .NET Foundation projects. We can also help ensure that your collaborators respect the open source licenses and trademarks of projects assigned to the .NET Foundation. Need advice on legal issues? We offer support there as well.
+
+## Marketing and communications
+
+The more people who know about .NET, the better for the ecosystem. We work with Amazon, JetBrains, Microsoft, Progress, and other industry leaders to improve your project's exposure within the community at large. Your success is our success - we'd love to see you achieve the accolades you deserve!
+
+## CLA management
+
+A Contributor License Agreement (CLA) ensures a project has the necessary rights to code contributions.
+
+Our automated CLA system has been working well for GitHub-based projects, with a very high rate of developers able to easily sign their Contribution License Agreements electronically. This service is offered with full automated notification to the project committers regarding the CLA status of a pull request author.
+
+## Secret management
+
+Get a secure vault to store secrets related to your project (social media credentials, website logins, etc.). Share credentials securely between trusted maintainers on your project, and enjoy the peace of mind that they're safely saved
+
+## Code signing
+
+Code signing protects against content tampering. A signature guarantees the assets haven't been modified since the asset was signed.
+
+We've implemented an automated service to Authenticode signing your release MSIs, EXEs, and NuGet packages with the .NET Foundation code signing certificate. The certificate is well known to Windows Smart Screen so that people can install your open-source application without warnings. It also allows for consuming applications knowing that the binary downloaded is the one created on your build server.
+
+## Hosting
+
+We provide hosting for a number of .NET Foundation project resources in Microsoft Azure. We have a .NET Foundation Office 365 account with SharePoint Online, Exchange, and more.
+
+## Tools
+
+If your maintainers need access to certain software to enable them to work more effectively on their open source projects, contact us and we can talk with vendors on your behalf. Microsoft has sponsored a number of MSDN subscriptions for .NET Foundation project contributors over the past 12 months.
+
+## Continuous integration
+
+The .NET Foundation has a dedicated Azure DevOps instance, which allows projects to use as many agents in Pipelines as they need and provide storage for CI feeds with Artifacts.
```

Comment:
**QUESTION:** How is it different from the regular Azure DevOps free tier for OSS projects?

Reply:
I guess the dedicated Azure DevOps instance has higher limits on CI minutes and Artifacts storage. Getting a little more detail here would be good.

Reply:
The Azure DevOps has higher limits on CI minutes/artifacts, but it's shared between multiple projects so when other projects have high usage there can be delays to projects running.